module/modifier-bucket/bucket-app.js

```diff
@@ -605,42 +588,43 @@ export class ModifierBucket extends Application {
   }
 
   /**
-   * @param {JQuery<HTMLElement>} html
+   * @param {JQuery<HTMLElement>} $html
    */
-  activateListeners(html) {
-    super.activateListeners(html)
+  activateListeners($html) {
+    super.activateListeners($html)
+
+    const html = $html[0]
 
-    html.find('#oned6dice').addClass('invisible')
-    html.find('#threed6dice').removeClass('invisible')
+    html.querySelector('#oned6dice').classList.add('invisible')
```

Comment:
Nice job updating this to html instead of jQuery, this is the kind of small step towards AppV2 I like.